geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/locks/DLockServiceCharacterizationTests.java

```diff
@@ -16,15 +16,25 @@
 package org.apache.geode.distributed.internal.locks;
 
 import static org.apache.geode.distributed.ConfigurationProperties.MCAST_PORT;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.catchThrowable;
 import static org.awaitility.Awaitility.await;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
```

Comment:
Again, a mixture of assertion libraries for no apparent reason other than new tests use the preferred assertj, but older tests not updated.

Oops, some of the new tests don't use assertj